src/GuestConfiguration/GuestConfiguration/Az.GuestConfiguration.psd1

```diff
@@ -104,14 +104,10 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '* Initial release of Az.GuestConfiguration module. Provides these two cmdlets.
-  * Get-AzVMGuestPolicyReport
-    * Provides compliance status of a VM in a resource group, compliance reasons.
-  * Get-AzVMGuestPolicyStatusHistory:
-    * Provides historical compliance statuses of a VM in a resource group, for a maximum of past 14 days.'
+        ReleaseNotes = 'General bug fixes and Cmdlet Get-AzVMGuestPolicyReport renamed to Get-AzVMGuestPolicyStatus to keep it consistent with the Azure portal and to avoid user confusion.'
 
         # Prerelease string of this module
-        # Prerelease = ''
+        #Prerelease = 'preview'
```

Comment:
I don't think this should be a preview package - generally we don't want  major version packages to be preview, as the '0' major version already allows for breaking changes.

Reply:
done